content/en/Concepts/environment.md

```diff
@@ -38,6 +38,13 @@ modules:
   # additional modules, ex: k8s-cluster
 ```
 
+## State Storage
+Unless you're using Opta's remote option, the state will be stored in your cloud's native bucket solution (e.g. S3, GCS, 
```

Comment:
what's the _remote_ option, can you add a link to the reference/doc for it please?

Reply:
local
I meant local whoops